index.d.ts

```diff
@@ -557,7 +562,7 @@ interface DCRBrowserDocumentData {
     page: string;
     site: string;
     CAPI: CAPIBrowserType;
-    NAV: NavType;
+    NAV: NavBrowserType;
```

Comment:
Having this as one `NAV` object doesn't really make sense anymore if the object only has 2 properties and isn't the same as `NAV` on the server